rest_framework/test.py

```diff
@@ -22,7 +23,7 @@
 
 
 def force_authenticate(request, user=None, token=None):
-    request._force_auth_user = user
+    request._force_auth_user = copy.deepcopy(user)
```

Comment:
I've ran into some *weird* [issues](https://github.com/philipn/django-rest-framework-filters/issues/171) recently with `deepcopy` in regards to the `request` object. Although it'd be unusual, it's not inconceivable that the request would be somehow accessible from the user via an authenticator or middleware, which `deepcopy` would pick up. I'm also hesitant about `deepcopy` here.